packages/@navikt/person-card/src/index.tsx

```diff
@@ -0,0 +1,32 @@
+import bem from '@navikt/nap-bem-utils';
+import * as React from 'react';
+import './indexStyles';
+import Card from './Card';
+
+const personCardCls = bem('person-card');
+
+export type Gender = 'male' | 'female' | 'unknown';
+
+export interface PersonCardData {
+    name: string;
+    fodselsnummer: string;
+    gender: Gender;
+    url: string;
+    isActive?: boolean;
+    renderMenuContent: () => React.ReactNode;
+}
+
+interface PersonCardProps {
+    personCardData: PersonCardData[];
+}
+
+const PersonCard = ({ personCardData }: PersonCardProps): JSX.Element => {
```

Comment:
Forstår ikke helt oppdelingen av `Card` og `PersonCard`. Card ser for meg ut til å være like bundet av Person-ting som PersonCard siden den forholder seg til både name, fodselsnummer og gender. Hvis vi skal ha en generell `Card`-komponent bør vel den bare lage en ramme rundt eller noe slikt, og overlate alt innholdet til f.eks. `PersonCard`?